blog/_posts/2016-09-20-julia-0.5-highlights.md

```diff
@@ -0,0 +1,726 @@
+In its simplest form, this looks something like:
+
+    julia> IntSet([1, 4, 9, 16, 25, 36, 49, 64])
+    IntSet([1, 4, 9, 16, 25, 36, 49, 64])
+
+In this input expression, an array of integers is passed to the `IntSet` constructor to create an object representing that set, which in this case happen to be small squares.
+Once constructed, the `IntSet` object no longer refers to the original array of integers.
+Instead, it uses a bitmask to efficiently store and operate on sets.
+It displays itself as you would construct it from an array of integers, but that's merely for convenience – there's no actual array anymore.
+
+Now, I'm a human (no blogbots here) and I find typing out even short sequences of perfect squares tedious and error prone – despite a math degree, I'm awful at arithmetic.
+It would be much easier to generate the values to go in this `IntSet` with an array comprehension:
+
+    julia> IntSet([k^2 for k = 1:8])
+    IntSet([1, 4, 9, 16, 25, 36, 49, 64])
+
+This comprehension produces the same array of integers that I typed above.
+Creating an array object is unnecessary, though – it would be even better to generate the desired squares just as they are inserted into the `IntSet` being constructed.
+Which, of course, is precisely what generator expressions allow:
+
+    julia> IntSet(k^2 for k = 1:8) # no temporary array!
+    IntSet([1, 4, 9, 16, 25, 36, 49, 64])
+
+More concise, just as clear, and (as we're about to see) faster – let's benchmark these two constructions:
+
+    julia> using BenchmarkTools
+
+    julia> @benchmark IntSet([k^2 for k = 1:8])
+    BenchmarkTools.Trial:
+      samples:          10000
+      evals/sample:     772
+      time tolerance:   5.00%
+      memory tolerance: 1.00%
+      memory estimate:  320.00 bytes
+      allocs estimate:  5
+      minimum time:     163.00 ns (0.00% GC)
+      median time:      199.00 ns (0.00% GC)
+      mean time:        245.18 ns (12.95% GC)
+      maximum time:     5.36 μs (92.47% GC)
+
+    julia> @benchmark IntSet(k^2 for k = 1:8)
+    BenchmarkTools.Trial:
+      samples:          10000
+      evals/sample:     925
+      time tolerance:   5.00%
+      memory tolerance: 1.00%
+      memory estimate:  160.00 bytes
+      allocs estimate:  3
+      minimum time:     114.00 ns (0.00% GC)
+      median time:      139.00 ns (0.00% GC)
+      mean time:        165.74 ns (11.48% GC)
+      maximum time:     4.82 μs (93.20% GC)
+
+I'm using the [`BenchmarkTools`](https://github.com/JuliaCI/BenchmarkTools.jl) package here instead of hand-rolled timing loops. `BenchmarkTools` has been carefully designed to avoid many of the common pitfalls of benchmarking code and to provide sound statistical estimates of how much time and memory your code uses.
```

Comment:
Maybe introduce this with the previous usage rather than after the second usage?